src/main/java/io/confluent/idesidecar/restapi/resources/RestProxyResource.java

```diff
@@ -115,5 +190,14 @@ private ClusterProxyContext createSRClusterContext(RoutingContext routingContext
         ClusterType.SCHEMA_REGISTRY
     );
   }
-
+  private ProxyContext createRBACProxyContext(RoutingContext routingContext){
+     return new ProxyContext(
+         RBAC_URI,
+         NO_HEADERS,
```

Comment:
Should we set `Content-Type: application/json` here instead of in `ControlPlaneAuthenticationProcessor`?